src/backend/base/langflow/schema/dataframe.py

```
@@ -178,3 +178,24 @@
         if not isinstance(other, DataFrame | pd.DataFrame):  # Non-DataFrame case
             return False
         return super().__eq__(other)
+
+    def to_data(self, v: Any) -> Data:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)